presto-orc/src/test/java/com/facebook/presto/orc/TestLongDictionaryProvider.java

```diff
@@ -237,9 +361,48 @@ public NodeId(int node, int sequence)
             this.sequence = sequence;
         }
 
+        @Override
+        public int hashCode()
+        {
+            return Objects.hash(node, sequence);
+        }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            if (o == this) {
+                return true;
+            }
+
+            NodeId other = (NodeId) o;
```

Comment:
Java equals require to handle the Object o being null and the Object not an instance of NodeId. 

Generally the second case after o == this is

if (!o instanceof NodeId) return false;

Reply:
My bad, I thought of this earlier but forgot to include it.